packages/flutter/lib/src/widgets/async.dart

```diff
@@ -333,19 +333,27 @@ typedef Widget AsyncWidgetBuilder<T>(BuildContext context, AsyncSnapshot<T> snap
 class StreamBuilder<T> extends StreamBuilderBase<T, AsyncSnapshot<T>> {
   /// Creates a new [StreamBuilder] that builds itself based on the latest
   /// snapshot of interaction with the specified [stream] and whose build
-  /// strategy is given by [builder].
+  /// strategy is given by [builder]. You can also specify the [initialData],
```

Comment:
The semantics of `initialData` should be clarified, cf. the general comments to this PR.